src/dcm/agent/plugins/builtin/unmount_volume.py

```diff
@@ -17,24 +17,48 @@
 import dcm.agent.plugins.api.base as plugin_base
 import dcm.agent.plugins.api.utils as plugin_utils
 import dcm.agent.utils as utils
+import logging
+import os
+import stat
 
+_g_logger = logging.getLogger(__name__)
+
+def _translate_ubuntu_device_id(deviceId):
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this parameter "deviceId" to match the regular expression ^[_a-z][a-z0-9_]*$. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.int.enstratius.com/coding_rules#rule_key=python%3AS117)